strr-web/pages/create-account-pre/[id].vue

```diff
@@ -132,7 +154,11 @@ const submit = () => {
       userLastName,
       hasSecondaryContact.value,
       propertyToApiType(formState.propertyDetails.propertyType),
-      ownershipToApiType(formState.propertyDetails.ownershipType)
+      ownershipToApiType(formState.propertyDetails.ownershipType),
```

Comment:
Why not pass the whole `formState.propertyDetails` object as param, instead of each prop separately?

Reply:
was staying consistent with what came before